addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -461,10 +461,9 @@ def _import_fill_invoice_line_values(self, tree, xpath_dict, invoice_line_form,
         rebate_node = tree.find(xpath_dict['rebate'])
         net_price_unit_node = tree.find(xpath_dict['net_price_unit'])
         if rebate_node is not None:
-            if net_price_unit_node is not None and gross_price_unit_node is not None:
-                rebate = float(gross_price_unit_node.text) - float(net_price_unit_node.text)
-            else:
-                rebate = float(rebate_node.text)
+            rebate = float(rebate_node.text)
+        elif net_price_unit_node is not None and gross_price_unit_node is not None:
+            rebate = float(gross_price_unit_node.text) - float(net_price_unit_node.text)
```

Comment:
Is this necessary? Can't we assume that if we have no rebate node, we have no rebate?

Reply:
I'm not sure. The rebate and gross unit price are optional while the net unit price is mandatory, so we could theoretically have a gross unit price and a net unit price with an "implicit" rebate. But it's quite a corner case.